$NetBSD: patch-aa,v 1.10 2005/11/28 14:45:39 rillig Exp $

Solaris needs __EXTENSIONS__ too, otherwise struct timeval is not
defined.

--- src/document.c.orig	2005-08-11 04:19:49.000000000 +0200
+++ src/document.c	2005-11-28 15:10:38.589617800 +0100
@@ -24,6 +24,7 @@
  * regarding the ctime_r() function
  * the problem is that it generates a compiler warning on Linux, lstat() undefined.. */
 #define _POSIX_C_SOURCE 200312L
+#define __EXTENSIONS__ 1
 
 #include <gtk/gtk.h>
 #include <gdk/gdkkeysyms.h> /* for the keyboard event codes */
